README.md

```diff
@@ -1123,16 +1123,28 @@ npm run lint
   const goodNewArray = [...originalArray];
   ```
 
-- [8.5](#8.5) <a name="8.5"></a> To convert from an array-like object to an array (for example, a `NodeList` returned by `document.querySelectorAll`, or a jQuery object), use `Array.from`.
+- [8.5](#8.5) <a name="8.5"></a> To convert from an array-like object to an array use `Array.from`.
```

Comment:
Can we just ditch this? I don't think it's particularly relevant advice and it makes the rule feel harder to understand than it really is (just use spread)